util/src/main/scala/Money.scala

```diff
@@ -88,15 +89,15 @@ case class Money private (amount: BigDecimal, currency: Currency) extends BaseMo
     "The scale of the given amount does not match the scale of the provided currency." +
     " - " + amount.scale + " <-> " + currency.getDefaultFractionDigits)
 
-  private val centFactor: Double = 1 / pow(10, currency.getDefaultFractionDigits)
+  private val centFactor: Double = Money.cachedCentFactor(currency.getDefaultFractionDigits).doubleValue
```

Comment:
compared to before, this new approach first uses `BigDecimal` before going the `double`.
Is there any impact?

Reply:
I was really unsure about this one.
My main assumption is that I expect the underlying `BigDecimal` to always be cached, so the bottom line is:

Is it better to compute the `Double` from scratch all the time or extract it from a cached `BigDecimal`?

In the absence of a benchmark to motivate the change properly, I tend more towards removing it for the time being.

WDYT?

Reply:
Let's try!

Reply:
I decided to remove it https://github.com/sphereio/sphere-scala-libs/pull/179/commits/537adcf255bf06943d9cc8ece61029bd1a71a5e9

Playing with the existing benchmarks it seems to either not make a difference or make things slower.